apps/admin_app/lib/admin_app_web/auth_pipeline/role_helper.ex

```diff
@@ -0,0 +1,22 @@
+defmodule AdminAppWeb.RoleHelper do
+  @moduledoc false
+
+  def load_role_permissions do
+    File.cwd!()
+    |> Path.join("/priv/role_manifest.yaml")
+    |> YamlElixir.read_from_file()
+  end
+
+  def check_role_permission("admin", _, _, _), do: true
+
+  def check_role_permission(_role, user_permissions, controller, action) do
```

Comment:
Very bad variable names :anger: 
I think, `user_permissions` are actually the permission `codes` of the  `_role`, pls call it permission_codes.
Why does this function return a list and not a boolean? Why is some of the work done [here](https://github.com/aviabird/snitch/pull/115/files#diff-73650273f8522716c1b9441ff671b509R38)?

Reply:
Have modified the function and names in PR fixes.